### PR TITLE
add kubernetes config

### PR DESCRIPTION
**Pull request by camallen**
setup the PRN maps API to work with kubernetes

### CODE REVIEW

File: kubernetes/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+              valueFrom:
+                secretKeyRef:
+                  name: prn-maps-api
+                  key: basic_auth_username
+            - name: BASIC_AUTH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: prn-maps-api
+                  key: basic_auth_password
+            - name: AWS_ACCESS_KEY_ID
```
Comment (adammcmaster) [inline kubernetes/deployment.yaml]: Fancy trying to set this up with an IAM role instead?

Comment (adammcmaster) [reply kubernetes/deployment.yaml]: It should just need an annotation like this: https://github.com/zooniverse/operations/blob/master/kubernetes/alb-ingress-controller.yaml#L30

Comment (adammcmaster) [reply kubernetes/deployment.yaml]: With the appropriate role name, obviously :)

Comment (camallen) [reply kubernetes/deployment.yaml]: Is that for all workers or just the deployment pods? I couldn't find any docs on this from 2 seconds of looking.

Comment (adammcmaster) [reply kubernetes/deployment.yaml]: The `kube2iam` service will assign the role to the pods, so each app we deploy can have it's own role now.

Comment (camallen) [reply kubernetes/deployment.yaml]: I've updated it with an IAM role...mind checking it again?

**Pull request merged**